sdk/core/System.ClientModel/api/System.ClientModel.net6.0.cs

```diff
@@ -7,6 +7,13 @@ public partial class ApiKeyCredential
         public static implicit operator System.ClientModel.ApiKeyCredential (string key) { throw null; }
         public void Update(string key) { }
     }
+    public abstract partial class AsyncResultCollection<T> : System.ClientModel.ClientResult, System.Collections.Generic.IAsyncEnumerable<T>
+    {
+        protected internal AsyncResultCollection() : base (default(System.ClientModel.Primitives.PipelineResponse)) { }
+        protected internal AsyncResultCollection(System.ClientModel.Primitives.PipelineResponse response) : base (default(System.ClientModel.Primitives.PipelineResponse)) { }
+        public static System.ClientModel.AsyncResultCollection<System.BinaryData> Create(System.ClientModel.Primitives.PipelineResponse response, string terminalEvent) { throw null; }
```

Comment:
Upon reflection, I don't think `AsyncResultCollection<BinaryData>` will work, because if we are only returning the data payloads from the SSE events, the end-user won't be able to see the event names, and in cases where they need to multiplex on different event types, they would not have the information needed to do that.

Reply:
Addressed.